backend/user/middleware.go

```diff
@@ -31,6 +31,9 @@ const (
 	RoleNonBCCMember = "non-bcc-members"
 )
 
+// DefaultRoles are default roles in the system.
+var DefaultRoles = []string{RolePublic, RoleRegistered, RoleBCCMember, RoleNonBCCMember}
```

Comment:
const?

Reply:
arrays cant be consts, as they are in nature dynamic

Reply:
oh ok.
but you can make fixed size arrays with [...]
https://stackoverflow.com/questions/13137463/declare-a-constant-array